profile/srap-profile.md

```diff
@@ -166,7 +175,9 @@ MARC relator: https://id.loc.gov/vocabulary/relators/edt
 
 Label: Editor
 
-A person or organization contributing to a resource by revising or elucidating the content, e.g., adding an introduction, notes, or other critical matter. An editor may also prepare a resource for production, publication, or distribution..
+A person or organization contributing to a resource by revising or elucidating the content, e.g., adding an introduction, notes, or other critical matter. An editor may also prepare a resource for production, publication, or distribution.
+
+Subproperty of: http://purl.org/dc/elements/1.1/contributor 
```

Comment:
Put this way, it looks like the profile may be declaring something that would normally be declared in the underlying vocabulary.

If it is useful to the readers and users of the profile to report this information in profile, perhaps it could be done in something that is more clearly marked as a "note", eg: "Note: This property is a subproperty of http://...".

Reply:
Change done.